src/Symfony/Bundle/FrameworkBundle/Resources/config/workflow.php

```diff
@@ -28,7 +28,7 @@
                 abstract_arg('events to dispatch'),
             ])
             ->abstract()
-            ->public()
+            ->public() // Deprecated since Symfony 5.3
```

Comment:
Tags from abstract definitions are not forwarded to the final definition. This is why we add the tag in FrameworkExtension.php, on the real workflow services definitions. This comment is just an indication.

The `container.private` tag feature does not support abstract definitions. If someone "extends" this definition and relies on its public availability, it will break in Symfony 6.0. We couId make an exception for `container.private` but I spoke with @nicolas-grekas about that and he judged that it was OK like that I agree that the probability that someone use those definitions (considering the number of abstract argument) is very low.

Reply:
you should add container.private instead of the comment, that will be a better documentation IMHO

Reply:
Alright :)